src/lib/services/addon-service.js

```diff
@@ -37,7 +37,15 @@ class AddonService {
 
         // Memoized private function
         this.fetchAddonConfigs = memoize(
-            () => addonStore.getAll({ enabled: true }),
+            async () => {
+                try {
+                    const addons = await addonStore.getAll({ enabled: true });
+                    return addons;
+                } catch (e) {
+                    this.logger.warn(`Could not get addons ${e}`);
+                    return [];
```

Comment:
I would actually argue that we should store the addon configurations and return the cached / stale once if we are unable to fetch the configs. Now we are ending up removing them, and not doing "addon" logic in the case the db is removed. It can lead to lost messages to a webhook (because of a temporary db-error).

Reply:
Good point.

Reply:
How would we store them, what does memoizee do if we don't return anything from a call instead of the empty list?

Reply:
Need to check the docs. I do not think memoizee do anything clever automatically.

Reply:
I see errors are not cached, but it also seems to void the cache for the key that fails. So it wasn't enough to simply not have error handling inside the memoizee wrapper

Reply:
Seems like it already does what we want. Does not cache a promise that errors, but does not crash the application either, it simply retries the wrapped function the next time we need the data. I reverted to how it looked on master for just this

Reply:
Which isn't what we want. We do want to keep the stale config. Hmm, let's discuss this tomorrow